src/main/java/de/schottky/core/UpgradableRangedWeapon.java

```diff
@@ -0,0 +1,69 @@
+package de.schottky.core;
+
+import com.github.schottky.zener.util.item.ItemStorage;
+import com.google.gson.JsonObject;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class UpgradableRangedWeapon extends UpgradableItem {
+    public static final Map<Material, UpgradableRangedWeapon> ALL_WEAPONS = new EnumMap<>(Material.class);
+
+    public final double arrowDamageMultiplier;
```

Comment:
Should be Modifier? (My original thought is: damage=damageModifier*damageMultiplier because I don't handle ranged weapon separately, I want to decrease the power of ranged weapon.)